spec/features/index_tests/user_views_posts_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature 'user views the home page', %{
+  As a user
+  I want to view a list of meetup posts
+  so I can meetup with other users
+  } do
+
+    before :each do
+      user = FactoryGirl.create(:user)
+    end
+    scenario 'unauthorized user views a list of posts' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.